src/assets/style/theme.js

```diff
@@ -0,0 +1,61 @@
+export const Breakpoints = {
```

Comment:
Much of this is not being used. Please pair this down to what we need as anything that is necessary can be added later.